Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -2242,6 +2255,20 @@ def panorama_get_url_category_command(url_cmd: str):
         context_urls = populate_url_filter_category_from_context(category)
         categories_dict[category] = list((set(categories_dict[category])).union(set(context_urls)))
 
+        score = calculate_dbot_score(url,category)
+        dbot_score = Common.DBotScore(
+            indicator=url,
+            indicator_type=DBotScoreType.URL,
+            integration_name='PAN-OS',
```

Comment:
i think this needs to match the integration brand/id - `Panorama`

Reply:
it is always PAN-OS throughout our integrations/playbooks etc so i would keep it for consistency